client_end_user/shared/src/commonMain/kotlin/presentation/search/SearchUiState.kt

```diff
@@ -0,0 +1,47 @@
+package presentation.search
+
+import domain.entity.Meal
+import domain.entity.Restaurant
+
+
+data class SearchUiState(
+    val query: String = "",
+    val meals: List<MealUiState> = emptyList(),
+    val restaurants: List<RestaurantUiState> = emptyList()
+)
+
+
+data class MealUiState(
+    val id: String = "",
+    val name: String = "",
+    val image: String = "",
+    val restaurantName: String = "",
+    val price: String = ""
+)
+
+data class RestaurantUiState(
+    val id: String = "",
+    val image: String = "",
+    val name: String = ""
+)
+
+//Mapper
+
+fun Meal.toUiState() = MealUiState(
+    id = id,
+    name = name,
+    image = image,
+    restaurantName = restaurantId,
```

Comment:
```suggestion
    restaurantName = restaurantName,
```
![image](https://github.com/TheChance101/beep-beep/assets/83292287/ed728b0e-9979-4a8f-80ca-bc9f03587b4e)

Reply:
I depend on real data not fake and in the backend we need to return the restaurant name and it's not done yet